Tanks/Assets/Scripts/Shell/LavaFieldScript.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LavaFieldScript : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public float m_LifeTime = 3f;
```

Comment:
Looks like this isn't used by other scripts. If it's not used, delete it. If it's needed and can be SerializedField private, make it so